roles/openjdk/tasks/ubuntu.yml

```diff
@@ -0,0 +1,14 @@
+---
+- name: Change keystore to JKS
+  become: yes
+  become_user: "root"
+  replace:
+    path: /etc/java-11-openjdk/security/java.security
+    regexp: 'keystore.type=pkcs12'
+    replace: 'keystore.type=jks'
+    backup: yes
+  notify:
+    - Delete CA certificates
```

Comment:
Could this be achieved just running the `update-ca-certificates` command (instead of first having to delete /etc/ssl/certs/java/cacerts then running the post install hook?)

Reply:
That did not seem to work on Ubuntu 16.04

Reply:
Cool. Do the commands need to be run immediately after ` /etc/java-11-openjdk/security/java.security`, or it's fine if they were done after OpenSRP has been set up?

Reply:
I have added flush handlers as requested